spark/src/main/java/org/apache/iceberg/spark/source/Reader.java

```diff
@@ -19,11 +19,13 @@
 
 package org.apache.iceberg.spark.source;
 
+import com.google.common.base.Function;
```

Comment:
We use `java.util.function` classes instead of the Guava ones.

Reply:
👍